### DISCUSSION TITLE
Problem with gcc installation trying to download non-existing gmp-6.1.2_2.catalina.bottle

### DISCUSSION THREAD

**Discussion by JohannesPfeifer** [Homebrew/discussions#1990] [Tap maintenance and Homebrew development] [answered]
In the Dynare support forum with increasingly get reports of users failing to install the program due to `gcc` on Big Sur trying to download `https://homebrew.bintray.com/bottles/gmp-6.1.2_2.catalina.bottle.1.tar.gz`, which does not exist. But we were unable to figure out where this wrong dependence comes from. See e.g. https://forum.dynare.org/t/installation-failure-with-dynare-4-6-4/18563/16. 

The probably relevant part of the `install.log` is 
```
==> Tapping homebrew/core
Cloning into '/Applications/Dynare/4.6.4/.brew/Library/Taps/homebrew/homebrew-core'...
Tapped 2 commands and 5746 formulae (6,063 files, 451.7MB).
+ HOMEBREW_CACHE=/Users/ashley/Library/Caches/Homebrew-Dynare
+ HOMEBREW_NO_AUTO_UPDATE=1
+ [[ -z '' ]]
+ HOMEBREW_NO_ANALYTICS=1
+ mv /Applications/Dynare/4.6.4/gcc.rb /Applications/Dynare/4.6.4/gmp.rb /Applications/Dynare/4.6.4/isl.rb /Applications/Dynare/4.6.4/libmpc.rb /Applications/Dynare/4.6.4/mpfr.rb /Applications/Dynare/4.6.4/.brew/Library/Taps/homebrew/homebrew-core/Formula
+ sudo -u ashley /Applications/Dynare/4.6.4/.brew/bin/brew install gcc -v --force-bottle
==> Downloading https://homebrew.bintray.com/bottles/gmp-6.1.2_2.catalina.bottle.1.tar.gz
/usr/bin/curl --disable --globoff --show-error --user-agent Homebrew/3.2.6-123-g0e63fb8\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 11.5.1\)\ curl/7.64.1 --header Accept-Language:\ en --fail --silent --retry 3 --location --remote-time --output /Users/ashley/Library/Caches/Homebrew/downloads/5aa09afe716bf4318e4482ea17edb1822a33ad814017ec7846346c8d59688ed2--gmp-6.1.2_2.catalina.bottle.1.tar.gz.incomplete https://homebrew.bintray.com/bottles/gmp-6.1.2_2.catalina.bottle.1.tar.gz
curl: (22) The requested URL returned error: 403 Forbidden
Error: Failed to download resource "gmp"
Download failed: https://homebrew.bintray.com/bottles/gmp-6.1.2_2.catalina.bottle.1.tar.gz
```

Any clues would be highly appreciated

**Comment by carlocab** [comment]
Bintray has shut down. Your users need to update their copy of `brew` with `brew update` in order to download from Homebrew's new bottle host (GitHub packages).

**Comment by SMillerDev** [comment, marked as answer]
> because Dynare uses its own local copy of Homebrew

I see this in the responses from the Dynare team, you're likely shipping an ancient brew version that doesn't work anymore since bintray shut down.

**Reply by JohannesPfeifer** [reply]
Thanks for the pointer. We will investigate this further.

**Reply by SMillerDev** [reply]
Just in case this isn't abundantly clear from the unknown errors you're seeing. Vendoring an external package manager is a really really really bad idea and in no way, shape or form a supported use-case for brew.
